scene/gui/tree.h

```diff
@@ -527,6 +530,9 @@ friend class TreeItem;
 
 	void set_value_evaluator(ValueEvaluator *p_evaluator);
 
+	void set_delayed_text_editor_enabled(bool enabled);
+	bool get_delayed_text_editor_enabled() const;
```

Comment:
Looking at the rest of the API, I think those names would be better: `set_delayed_text_editor(bool enabled)` and `is_delayed_text_editor_enabled()` (or directly `is_delayed_text_editor` if it's clear enough).
See e.g. http://docs.godotengine.org/en/latest/classes/class_httpclient.html?highlight=enabled#class-httpclient-is-blocking-mode-enabled or http://docs.godotengine.org/en/latest/classes/class_graphedit.html?highlight=enabled#class-graphedit-is-right-disconnects-enabled